autogpt/plugins.py

```diff
@@ -18,6 +20,23 @@
 from autogpt.models.base_open_ai_plugin import BaseOpenAIPlugin
 
 
+def install_plugin_dependencies():
```

Comment:
should this call into the file in scripts or does this replace it?

Reply:
Good call. Didn't realize `scripts` was a module. Updated